scripts/geopmpy/launcher.py

```diff
@@ -1640,19 +1640,19 @@ def preload_option(self):
 
     def quiet_option(self):
         """
-        Returns a list containing the -q option for aprun.
+        Returns a list containing the ``-q`` option for ``aprun``.
         """
         return ['-q']
 
 
 def main():
     """
-    Main routine used by geopmlaunch wrapper executable.
+    Main routine used by ``geopmlaunch`` wrapper executable.
     This function creates a launcher from the factory and
-    calls the run method.  If help was requested on the command line
+    calls the ``run()`` method.  If help was requested on the command line
```

Comment:
run() could also be a python reference link.

Reply:
Thanks for teaching me how to do the python reference links.